pandas/tests/frame/test_missing.py

```diff
@@ -571,8 +585,12 @@ def test_interp_various(self):
         assert_frame_equal(result, expected, check_dtype=False)
 
         result = df.interpolate(method='quadratic')
-        expected.A.loc[3] = 2.82533638
-        expected.A.loc[13] = 6.02817974
+        if _is_scipy_ge_0190:
+            expected.A.loc[3] = 2.82150771
+            expected.A.loc[13] = 6.12648668
+        else:
+            expected.A.loc[3] = 2.82533638
+            expected.A.loc[13] = 6.02817974
```

Comment:
@ev-br main issue is that, somehow `quadratic` and `cubic` interpolation results revert back to pre 0.19 behavior.